theano/tensor/nnet/corr_gemm.c

```diff
@@ -162,7 +162,7 @@ PyArrayObject* corrMM(PyArrayObject* bottom,
                 "CorrMM shape inconsistency:\n"
                 "  bottom shape: %%d %%d %%d %%d\n"
                 "  weight shape: %%d %%d %%d %%d\n"
-                "  top shape: %%d %%d %%d %%d (expected %%d %%d %%d %%d)\n",
+                "  top shape: %%ld %%ld %%ld %%ld (expected %%d %%d %%d %%d)\n",
```

Comment:
Because the ones I changed are already `long`.  The others are ints (that should maybe be changed to longs).